gef-ui/server/utils/helpers.test.js

```diff
@@ -615,3 +616,9 @@ describe('getFacilitiesAsArray', () => {
     expect(getFacilitiesAsArray(MOCK_FACILITY)).toEqual(expected);
   });
 });
+
+describe('coverDatesConfirmed', () => {
+  it('Should return FALSE as one of the facility\'s cover date has not been confiemed', () => {
+    expect(coverDatesConfirmed(MOCK_FACILITY)).toEqual(false);
+  });
+});
```

Comment:
Just to confirm did you chat with Liz about this - i.e should they be able to submit if e.g only one of the facilities has cover dates confirmed?

Wondering because this was a requirement in BSS

Reply:
Confirmed the requirement with Liz, we are all good.